Modules/Microsoft365DSC/DSCResources/MSFT_IntuneAppleMDMPushNotificationCertificate/MSFT_IntuneAppleMDMPushNotificationCertificate.schema.mof

```diff
@@ -0,0 +1,25 @@
+[ClassVersion("1.0.0.0"), FriendlyName("IntuneAppleMDMPushNotificationCertificate")]
+class MSFT_IntuneAppleMDMPushNotificationCertificate : OMI_BaseResource
+{
+    [Key, Description("The name of the Apple Identifier.")] String AppleIdentifier;
+    [Write, Description("The Apple Push notification certificate.")] String Certificate;
+
+    [Write, Description("The unique identifier for an entity. Read-only.")] String Id;
+    [Write, Description("The Topic Identifier. Also known as Subject Id.")] String TopicIdentifier;
+    [Write, Description("The upload status of the Apple Push notification certificate.")] String CertificateSerialNumber;
+
+    [Write, Description("The last modified DateTime of the Apple Push notification certificate.")] DateTime LastModifiedDateTime;
+    [Write, Description("The expiration DateTime of the Apple Push notification certificate.")] DateTime ExpirationDateTime;
+
+    [Write, Description("The upload status of the Apple Push notification certificate.")] String CertificateUploadStatus;
+    [Write, Description("The failure reason of upload of the Apple Push notification certificate.")] String CertificateUploadFailureReason;
```

Comment:
Should also be removed, are not associated with the resource in a practicable manner.